Demo/UI Tests/American Express UI Tests/AmericanExpress_UITests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+class AmericanExpress_UITests: XCTestCase {
+
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        app = XCUIApplication()
+        app.launchArguments.append("-EnvironmentSandbox")
+        app.launchArguments.append("-ClientToken")
+        app.launchArguments.append("-Integration:BraintreeDemoAmexViewController")
+        app.launch()
+
+        waitForElementToAppear(app.buttons["Valid card"])
+    }
+
+    func testValidCard_receivesRewardsBalance() {
+        app.buttons["Valid card"].tap()
+        waitForElementToAppear(app.buttons["45256433 Points, 316795.03 USD"], timeout: 10)
```

Comment:
Do these `waitForElementToAppear` statements need to be wrapped in some sort of assertion?  It's unclear if the tests will actually fail or just return false if the wait times out.

Reply:
The test will fail if the element isn't found in the given amount of time. [`waitForElementToAppear`](https://github.com/braintree/braintree_ios/blob/bf20534a349502c74603eeba8b61e42ba4801b47/Demo/UI%20Tests/Helpers/BTUITest.swift#L4) calls XCTest's `waitForExpectations` method, which fails the test if the expectation isn't fulfilled.

Reply:
Ah ok thanks for the explanation!